SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -245,6 +245,10 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Logging", meta = (DisplayName = "Worker Log Level"))
 	TEnumAsByte<ESettingsWorkerLogVerbosity::Type> WorkerLogLevel;
 
+	/** Enable/Disable logging to Spatial */
+	UPROPERTY(EditAnywhere, config, Category = "Logging", meta = (DisplayName = "Disable Log to Spatial"))
+	bool bDisableLoggingToSpatial;
```

Comment:
We've had discussions in the past where `enable = false, disable = true` is a bad design pattern for UX (vs `enable = true, disable = false`). Might be worth swapping this logic to be `bEnableLoggingToSpatial` and have it default to be `true`.